swayfmt/src/utils/map/comments.rs

```diff
@@ -171,7 +171,7 @@ fn add_comments(
 }
 
 // A CommentWithContext is the Comment and the offset before it. The offset can be between the (from) item we searched for this comment or from the last comment inside range
-type CommentWithContext = (Comment, String);
+type CommentWithContext = (Comment, String, String);
```

Comment:
I highly doubt this would be the case but: if ^this change results too many `Option`s popping up in function signatures we can also use an enum like:

```rust
enum PostContext {
  None,
  Some(String)
}
```